web/common/routes.js

```diff
@@ -259,13 +292,14 @@ export const ROUTES = [
     path: "/logout",
     label: "Déconnexion",
     accessible: () => true,
-    menuItemFilter: () => false,
+    menuItemFilter: ({ controllerInfo }) => !!controllerInfo?.id,
     component: Logout
   },
   {
     label: "Mon compte",
     path: "",
-    accessible: ({ userInfo }) => !!userInfo.id,
+    accessible: ({ userInfo, controllerInfo }) =>
+      !!userInfo?.id && !controllerInfo?.id,
```

Comment:
Here, it means _userInfo.id is defined to a truethy value and controllerInfo.id is falsey_. I think this is correct and the issue may occur when we use only `if (userInfo?.id)` instead of `if (!!userInfo?.id)`.